### DISCUSSION TITLE
Please allow me to change the Star list to private.

### DISCUSSION THREAD

**Discussion by happy-spark** [community/community#35655] [Other Feature Feedback, Questions, & Ideas] [answered]
Please allow me to change the Star list to private.

**Comment by jostasik** [comment, marked as answer]
I agree it would be nice to have control over the privacy of our star list. Privatizing lists has been a very [commonly requested feature.](https://github.com/community/community/discussions/8766#discussioncomment-1909373) It is definitely on the employees' radar.

My advice would be to bookmark them for now. Another option, if you need to save the code is to clone a repo and upload the source into a new private repo.

**Comment by pataquets** [comment]
See #8626
